FtcRobotController/src/main/java/org/firstinspires/ftc/robotcontroller/external/samples/HardwarePushbot.java

```diff
@@ -82,17 +81,15 @@ public void init(HardwareMap ahwMap) {
      * @param periodMs  Length of wait cycle in mSec.
      */
     public void waitForTick(long periodMs) {
-
         long  remaining = periodMs - (long)period.milliseconds();
 
         // sleep for the remaining portion of the regular cycle period.
-        if (remaining > 0) {
+        if (remaining > 0)
             try {
                 Thread.sleep(remaining);
             } catch (InterruptedException e) {
                 Thread.currentThread().interrupt();
             }
-        }
```

Comment:
Speaking for myself, I am rather agnostic on everything in this diff except for removal of the braces defining the compound statement of the if clause.  I think it's worth pointing out that if clauses originally written with a single statement and omission of braces are a well known source of potential maintainability issues.   So much so, that every company I've ever worked for has banned this practice.  And so, if an author chooses the style which is less likely to result in inadvertent bugs as software evolves, I don't think it's prudent to arbitrarily change that.

On a broader topic, style itself has generated endless wars.   Again, speaking only for myself, unless something is grossly inconsistent I'll generally defer to the author and resist reformatting someone's software.  Jeff Atwood, one of the co-founders of StackExchange had a pretty good piece on this topic a number of years ago, https://blog.codinghorror.com/death-to-the-space-infidels/